tests/integration/test_logcollector/test_command_monitoring/test_command_execution_freq.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import sys
+from datetime import timedelta, datetime
+
+from wazuh_testing import logger
+from wazuh_testing.tools import get_service, monitoring
```

Comment:
```suggestion
from wazuh_testing.tools import monitoring
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1257/commits/1e33fb8c458aeb8a738e22706713ec98fbf25703